src/System.Collections.Concurrent/tests/project.json

```diff
@@ -14,7 +14,8 @@
     "xunit.abstractions.netcore": "1.0.0-prerelease",
     "xunit.assert": "2.0.0-beta5-build2785",
     "xunit.core.netcore": "1.0.1-prerelease",
-    "xunit.runner.visualstudio": "0.99.9-build1021"
+    "xunit.runner.visualstudio": "0.99.9-build1021",
+    "xunit.netcore.extensions": "1.0.0-prerelease-*"
```

Comment:
@ericstj is there any reasons to avoid "*" wildcard versioning with the current build tools? Or are we good to use it now?